src/routes/graph/update-pin-groups.ts

```diff
@@ -19,8 +17,6 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
   method: 'post',
   path: '/update-pin-groups/:hashId',
   body: Joi.object().keys({
-    hashIds: Joi.array().items(Joi.string()).required(),
```

Comment:
Hehe, the ordering was not the same here, you removed `hashIds` instead of `gridHashIds` (line 28, now 24)

Reply:
I should sleep more.
Updated